.travis.yml

```diff
@@ -33,43 +33,19 @@ matrix:
         - mysql-server-5.6
         - mysql-client-core-5.6
         - mysql-client-5.6
-  - php: "nightly"
-    env: WP_VERSION=master
   - php: "5.6"
     env: WP_VERSION=4.5 WP_TRAVISCI="gulp travis:js"
   - php: "5.6"
     env: WP_VERSION=4.5 WP_TRAVISCI="npm run test-client"
   - php: "5.2"
```

Comment:
If we're going to be testing 5.2, let's test it with WordPress stable, not master, what do you say?